src/containers/TopNav/TopNavButton/index.tsx

```diff
@@ -2,20 +2,20 @@ import React, {FC} from 'react';
 import clsx from 'clsx';
 import {Link} from 'react-router-dom';
 
-import './TopNavText.scss';
+import './TopNavButton.scss';
 
 interface ComponentProps {
   buttonText: string;
   className?: string;
   toUrl: string;
 }
 
-const TopNavText: FC<ComponentProps> = ({buttonText, className, toUrl}) => {
+const TopNavButton: FC<ComponentProps> = ({buttonText, className, toUrl}) => {
   return (
     <Link to={toUrl}>
```

Comment:
I think we should add `tabIndex={-1}` here on the Link.

Right now, both `a` and `button` elements are focusable, this causes me to focus two times on 'Create Account' and 'Sign In' each.